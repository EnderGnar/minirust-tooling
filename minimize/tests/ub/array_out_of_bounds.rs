fn main() {
    let x = [1, 2];
    let i = black_box(2);
    let _y = x[i];
}

fn black_box(i: usize) -> usize { i }
