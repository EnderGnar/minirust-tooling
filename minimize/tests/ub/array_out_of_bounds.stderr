UB: out-of-bounds array access
